lib/Runtime/Library/ScriptFunction.h

```diff
@@ -45,6 +45,7 @@ namespace Js
     public:
         ScriptFunction(FunctionProxy * proxy, ScriptFunctionType* deferredPrototypeType);
         static bool Is(Var func);
+        static BOOL Test(JavascriptFunction *func) { return func->GetFunctionInfo()->HasBody(); }
```

Comment:
Vtable works for sure. That's the problem. Intention with Test is not the same what "Is" provides